python/marvin/contrib/vacs/mangagalaxyzoo.py

```diff
@@ -0,0 +1,90 @@
+# !usr/bin/env python
+# -*- coding: utf-8 -*-
+#
+# Modified from mangahi.py by Karen Masters. 
+
+from __future__ import print_function, division, absolute_import
+
+import numpy as np
+import astropy
+import astropy.units as u
+import marvin.tools
+from marvin.tools.quantities.spectrum import Spectrum
+from marvin.utils.general.general import get_drpall_table
+from marvin.utils.plot.scatter import plot as scatplot
```

Comment:
The imports of `get_drpall_table` and `scatplot` do not get used.  If you are not intending to use them, you can safely remove these

Reply:
The VAC is just a catalogue, and we just want Marvin users to be able to access it. Agreed no need for anything else. I'll remove those lines.